phys2bids/interfaces/txt.py

```diff
@@ -98,39 +98,45 @@ def process_labchart(channel_list, chtrig, header=[]):
     # get frequency
     # check header has some length
     if len(header) == 0:
-        raise AttributeError('Files without header are not supported yet')
+        raise AttributeError("Files without header are not supported yet")
     interval = header[0][1].split(" ")
     # check the interval is in some of the correct labchart units
-    if interval[-1] not in ['hr', 'min', 's', 'ms', 'µs']:
-        raise AttributeError(f'Interval unit "{interval[-1]}" is not in a valid LabChart'
-                             'time unit, this probably means your file is not in Labchart format')
+    if interval[-1] not in ["hr", "min", "s", "ms", "µs"]:
+        raise AttributeError(
+            f'Interval unit "{interval[-1]}" is not in a valid LabChart'
+            "time unit, this probably means your file is not in Labchart format"
+        )
     # check if interval is in seconds, if not change the units to seconds
-    if interval[-1] != 's':
-        LGR.warning('Interval is not in seconds. Converting its value.')
-        if interval[-1] == 'hr':
+    if interval[-1] != "s":
+        LGR.warning("Interval is not in seconds. Converting its value.")
+        if interval[-1] == "hr":
             interval[0] = float(interval[0]) * 3600
-            interval[-1] = 's'
-        elif interval[-1] == 'min':
+            interval[-1] = "s"
+        elif interval[-1] == "min":
             interval[0] = float(interval[0]) * 60
-            interval[-1] = 's'
-        elif interval[-1] == 'ms':
+            interval[-1] = "s"
+        elif interval[-1] == "ms":
             interval[0] = float(interval[0]) / 1000
-            interval[-1] = 's'
-        elif interval[-1] == 'µs':
+            interval[-1] = "s"
+        elif interval[-1] == "µs":
             interval[0] = float(interval[0]) / 1000000
-            interval[-1] = 's'
+            interval[-1] = "s"
     else:
         interval[0] = float(interval[0])
     # get units
     range_list = header[5][1:]
     orig_units = []
     for item in range_list:
-        orig_units.append(item.split(' ')[1])
-    units = ['s', ]
+        orig_units.append(item.split(" ")[1])
+    units = [
+        "s",
+    ]
```

Comment:
I know why black is formatting this list in this way throughout the whole file, but it's awful. can we add a #fmt: off (or whatever it was) here?

Reply:
This is fixed by just not including the trailing commas in the lists (not sure why they're there). `black` won't reformat `units = ["s"]`, so I'll just revert those.

Reply:
After digging a bit, it turns out that trailing commas are there to make lists easily extensible, so it actually makes sense to expand any list with a comma at the end to multiple lines because you expect that list to grow and you want minimal diffs when that happens. In any case, these lists aren't likely to grow, so I just removed the trailing commas and put them on single lines.